ui/src/core/index.ts

```diff
@@ -126,19 +126,39 @@ export function generateCore() {
 			*/
 
 			const accessor = parseAccessor(key);
+			let lastElementIndex = accessor.length - 1
 			let stateRef = userState.value;
-			for (let i = 0; i < accessor.length - 1; i++) {
+			
+			// Check if the accessor is meant for deletion.
```

Comment:
Let's always use an accessor character, not just deletion or addition. Otherwise I think we'd potentially have trouble when someone mutates something called `-myvar`. Plus it looks a bit less hacky at least to me.

Feel free to use the same for addition and mutation (+) or use separate e.g. + for addition, ~ for mutation.